src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+import {LocalizedLabels, LocalizedSteps, registerLocalization} from '../localization';
+
+const uiLabels: LocalizedLabels = {
+  'Angular Update Guide': 'Hướng dẫn Update Angular',
+  'Select the options matching your project:': 'Tuỳ chỉnh những lựa chọn cho phù hợp với dự án của bạn:',
+  'Angular Versions': 'Phiên bản Angular',
+  From: 'Từ',
+  To: 'Đến',
+  Warning: 'Chú ý',
+  'Plans for releases after the current major release are not finalized and may change. These recommendations are based on scheduled deprecations.':
+    'Đối với những phiên bản Angular được phát hành sau phiên bản chính thức hiện tại, kế hoạch và dự định cho những phiên bản này chưa được thông qua chính thức và có thể sẽ thay đổi. Những gợi ý sau dựa trên những chỉnh sửa đã được thống nhất.',
```

Comment:
deprecations != chỉnh sửa. Use "tính năng không còn được cập nhật"?

Reply:
Totally agree with the meaning isn't the same. But putting "tính năng không còn được cập nhật đã được thống nhất" doesn't flow really well imho. Do you have any other suggestions?